src/pages/filter/ui/modal-edit/modal-edit.store.ts

```diff
@@ -47,22 +59,64 @@ class ModalEditStore {
     ].length
   }
 
+  public get currentStepGroups(): string[] {
+    return dtreeStore.currentStepGroups
+  }
+
   public get problemGroups(): string[] {
-    let attrData: any
+    let problemGroups: string[] = []
 
     const subGroups = Object.values(
       getQueryBuilder(dtreeStore.startDtreeStat['stat-list']),
     )
 
+    subGroups.forEach(subGroup => {
+      subGroup.find((item, currNo) => {
+        if (item.name === this.groupName) {
+          problemGroups = subGroup[currNo].family
+        }
+      })
+    })
+
+    return problemGroups
+  }
+
+  public get approxModes() {
```

Comment:
And here you can also iterate over `stat-list`.
By the way, i think that you can create getter for getting current item from `stat-list`